config/routes.rb

```diff
@@ -11,7 +11,10 @@
   namespace :assessor_interface, path: "/assessor" do
     root to: redirect("/assessor/applications")
 
-    resources :application_forms, path: "/applications", only: %i[index show]
+    resources :application_forms, path: "/applications", only: %i[index show] do
+      get "assign-assessor", to: "assessor_assignments#new"
+      post "assign-assessor", to: "assessor_assignments#create"
```

Comment:
```suggestion
      resource :assessor_assignment, only: %i[new create], on: :member
```

I tend to prefer using as it abstracts away the routes and does a few things automatically, but I don't feel strongly either way.

Reply:
Happy to change but I can't get it to keep the URLs that we specified in the card doing it that way. We'd have to have `assign-assessor/new` If we're going to do that we may as well go with `assessor-assignments`. I don't really mind that, it just seemed to read better as `assign-assessor`

Reply:
Discussed offline. Leaving as is for now to preserve the URL.